models/ens/ens_ethereum_schema.yml

```diff
@@ -305,4 +305,29 @@ models:
               values: ['Queued', 'Active', 'Executed', 'Canceled','Defeated']
       - &description
         name: description
-        description: "Description of the proposal"
\ No newline at end of file
+        description: "Description of the proposal"
+
+  - name: ens_set_name_latest
+    meta:
+      blockchain: ethereum
+      project: ethereum_name_service
+      contributors: sankinyue
+    config:
+      tags: ['ethereum','ens','ethereum_name_service','ethereumnameservice','setname','latest']
+    description: >
+      View the latest ENS set name records
+    columns:
+      - name: last_block_time
```

Comment:
You can add a uniqueness test here.

Reply:
I think this will cause conflicts, such as different addresses set they ens name at the same time, 
which may happen occasionally

Reply:
Something like this to test unique_key works in incremental updates:
https://github.com/duneanalytics/spellbook/blob/main/models/blur/ethereum/blur_ethereum_schema.yml#L14